modules/task/execution.go

```diff
@@ -133,6 +133,15 @@ func Register(fn interface{}) error {
 	return nil
 }
 
+// MustRegister registers a function for an async task or panics
+// Since Task registration is performed in main() we want to fail-fast and
+// reduce the amount of error-checking
+func MustRegister(fn interface{}) {
```

Comment:
This assumes that Register never panics, which I think it is(e.g. when GlobalBackend() is nil?)
Could you add a todo, to make Register panic free then?